teerex/ias-verify/src/collateral.rs

```diff
@@ -111,12 +113,41 @@ pub struct EnclaveIdentity {
 	miscselect_mask: String,
 	attributes: String,
 	attributes_mask: String,
-	mrsigner: String,
+	#[serde(deserialize_with = "decode_mrsigner")]
+	#[serde(serialize_with = "encode_mrsigner")]
+	mrsigner: [u8; 32],
```

Comment:
I added a type alias for mrsigner and mrenclave. For mrsigner the situation is sometimes a bit fuzzy because it is also used as a public key in the blockchain domain (for example to sign an extrinsic).